Client/Frontend/Shields/ShieldsView.swift

```diff
@@ -38,31 +39,35 @@ extension ShieldsViewController {
         override init(frame: CGRect) {
             super.init(frame: frame)
             
-            addSubview(scrollView)
-            scrollView.addSubview(stackView)
-            
-            scrollView.snp.makeConstraints {
-                $0.edges.equalTo(self)
-            }
-            
-            scrollView.contentLayoutGuide.snp.makeConstraints {
-                $0.width.equalTo(self)
-            }
-            
-            stackView.snp.makeConstraints {
-                $0.edges.equalTo(scrollView.contentLayoutGuide)
-            }
-            
             stackView.addArrangedSubview(shieldOverrideControl)
             stackView.addArrangedSubview(overviewStackView)
             stackView.addArrangedSubview(shieldsContainerStackView)
+            
+            addSubview(scrollView)
+            scrollView.addSubview(stackView)
         }
         
         @available(*, unavailable)
         required init?(coder aDecoder: NSCoder) {
             fatalError()
         }
         
+        override func layoutSubviews() {
+            super.layoutSubviews()
+            
+            scrollView.snp.remakeConstraints {
+                $0.edges.equalToSuperview()
+            }
+            
+            scrollView.contentLayoutGuide.snp.remakeConstraints {
+                $0.left.right.equalTo(self)
+            }
+            
+            stackView.snp.remakeConstraints {
+                $0.edges.equalToSuperview()
```

Comment:
This works, and it will also work if constrained to the contentLayoutGuide.

IE: `scrollView.frameLayoutGuide.constrain(to: self)` and `stackView.constrain(to: scrollView.contentLayoutGuide)`.

But your code works so it's fine :)